apis/low_level_debug/src/tests.rs

```diff
@@ -0,0 +1,80 @@
+use super::*;
+use libtock_platform::ErrorCode;
+use libtock_unittest::{command_return, fake, ExpectedSyscall};
+
+#[test]
+fn driver_check() {
+    // Create a new fake kernel for the current thread, replacing avy previous one.
+    let kernel = fake::Kernel::new();
+    let driver = fake::LowLevelDebug::new();
+    kernel.add_driver(&driver);
+
+    assert!(LowLevelDebug::<fake::Syscalls>::driver_check().is_success());
```

Comment:
Please also test the case where the driver is not present, which can be done before the `kernel.add_driver` call.

Reply:
Done.